server/controllers/match/placementRequestsController.test.ts

```diff
@@ -42,7 +44,8 @@ describe('PlacementRequestsController', () => {
   })
 
   describe('index', () => {
-    it('should render the placement requests template', async () => {
+    it('should render the placement requests template when cruDashboardDisabled is false', async () => {
+      config.flags.cruDashboardDisabled = false
```

Comment:
I think it would be worthwhile to reset the state of the `cruDashboardDisabled` boolean in an afterEach as it will hang around after the tests have run which could cause headaches